CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ## 0.10.4
-* Implement `highlight` in `SourceMapFileSpan`.
-* Require version `^1.3.0` of `source_span`.
+ * Implement `highlight` in `SourceMapFileSpan`.
+ * Require version `^1.3.0` of `source_span`.
```

Comment:
Why was a space added here?

Reply:
For consistency with entries below.
Not that those are consistent anyway, if you go further down, so I'll just remove the spaces again.